### DISCUSSION TITLE
Lifepo4 cell undervoltage

### DISCUSSION THREAD

**Discussion by pieterjanvz** [KG-Development/SmartBMSUtility#8] [General]
I try to set cell undervoltage higher than the default 2500 mV for lifepo4 , but I can’t set it lower than 3000 mV through the app ,

**Comment by NeariX67** [comment]
Thanks for the info. Will be fixed on the next update

**Comment by pieterjanvz** [comment]
Thanks , another thing I found , I can not set the total battery capacity , after I set it it was at “0”  now have set it through jbd tool 1.6 on PC

**Reply by NeariX67** [reply]
What did you try to set it to?

**Comment by pieterjanvz** [comment]
Value was automatically updated by bms to more than 100Ah , I set is back to 100000mAh

**Reply by NeariX67** [reply]
I got this fixed. It will be in the next update. Max capacity will be 655350 mAh instead of 65530 mAh

**Reply by pieterjanvz** [reply]
Thanks , I found a new thing , if i change a setting with your tool , say LVC everything is working fine , until I restart the bms (by pulling the balance connector) then the setting is reverted to the old setting. If I do the same with jbd tool this does not happen.
